tests/amplitude/test_yaml_helicity.py

```diff
@@ -19,7 +19,8 @@
 SCRIPT_PATH = dirname(realpath(__file__))
 
 
-def create_amplitude_generator():
+@pytest.fixture(scope="module")
+def amplitude_generator():
```

Comment:
why not rebase?

Reply:
https://github.com/ComPWA/expertsystem/pull/203#issuecomment-674417427